lib/python/Components/Converter/MovieInfo.py

```diff
@@ -46,6 +49,10 @@ def getText(self):
 			elif self.type == self.MOVIE_REC_SERVICE_NAME:
 				rec_ref_str = info.getInfoString(service, iServiceInformation.sServiceref)
 				return ServiceReference(rec_ref_str).getServiceName()
+			elif self.type == self.MOVIE_NAME:
+				if (service.flags & eServiceReference.flagDirectory) == eServiceReference.flagDirectory:
+					return service.getPath()
```

Comment:
This will give you the full path of the directory.
Are you sure that you wanna have this?